config/lib/jwtAuthentication.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var config = require('../config'),
+  jwt = require('jsonwebtoken');
+
+// build token auth service
+var tokenAuth = {
+  signToken: signAuthToken
+};
+
+// export the token auth service
+module.exports = tokenAuth;
```

Comment:
I'd rather have this at the bottom because that's where we have most of the  exports. Also, you can just assign the object rather than create one.